README.md

```diff
@@ -244,6 +245,28 @@ authClient.verifyRecoveryToken({
 });
 ```
 
+## [webfinger(options)]
+
+Calls the Webfinger API and gets a response.
+
+- `resource` - URI that identifies the entity whose information is sought, currently only acct scheme is supported (e.g acct:dade.murphy@example.com)
+- `rel` - Optional parameter to request only a subset of the information that would otherwise be returned without the "rel" parameter
+- `requestContext` - A parameter that provides Webfinger the context of that which the user is trying to access, such as the path of an app
+
+```javascript
+authClient.webfinger({
+  resource: 'acct:john.joe@example.com',
+  rel: 'okta:idp',
+  requestContext: 'https://example.okta.com'
```

Comment:
it does not need to pass the requestContext if user just tries to access the homepage, an example of requestContext will be "/home/dropbox/0oa16630PzpWKeWrH0g4/121"

Reply:
👍 Will change